src/Compiler/Microsoft.AspNetCore.Razor.Language/test/CodeGeneration/DesignTimeNodeWriterTest.cs

```diff
@@ -457,6 +457,52 @@ Render Children
             ignoreLineEndingDifferences: true);
     }
 
+    [OSSkipConditionTheory(new[] { "Linux", "osx" })]
+    [InlineData("test.cshtml", "test.cshtml")]
+    [InlineData("pages/test.cshtml", "pages\\test.cshtml")]
+    [InlineData("pages\\test.cshtml", "pages\\test.cshtml")]
+    [InlineData("c:/pages/test.cshtml", "c:\\pages\\test.cshtml")]
+    [InlineData("c:\\pages\\test.cshtml", "c:\\pages\\test.cshtml")]
+    [InlineData("c:/pages with space/test.cshtml", "c:\\pages with space\\test.cshtml")]
+    [InlineData("c:\\pages with space\\test.cshtml", "c:\\pages with space\\test.cshtml")]
+    public void LinePragma_Is_Adjusted_On_Windows(string fileName, string expectedFileName)
+    {
+        // Arrange
+        var writer = new DesignTimeNodeWriter();
+        var context = TestCodeRenderingContext.CreateDesignTime();
+
+        var node = new CSharpExpressionIntermediateNode()
+        {
+            Source = new SourceSpan(fileName, 0, 0, 0, 3),
+        };
+        var builder = IntermediateNodeBuilder.Create(node);
+        builder.Add(new IntermediateToken()
+        {
+            Content = "i++",
+            Kind = TokenKind.CSharp,
+        });
+
+        // Act
+        writer.WriteCSharpExpression(context, node);
+
+        // Assert
+        var csharp = context.CodeWriter.GenerateCode();
```

Comment:
nit: I would say this line should be under `// Act`.

Reply:
I'd honestly rather remove every single `// act/assert` comment in this codebase. They're not useful and just clutter the screen.

Reply:
💯 

Cargo cult programming at its best